stack/base/llama-stack-distribution.yaml

```
@@ -26,6 +26,6 @@ spec:
       name: llama-stack
       port: 8321
     distribution:
-      image: quay.io/opendatahub/llama-stack:odh
+      image: registry.redhat.io/rhoai/odh-llama-stack-core-rhel9@sha256:41e1976d1810332c87a3c3bd90e2be5dff5f490cf29f769954406f17e3a6d10a
```

This is the recommended setting for RHOAI 2.23:
```suggestion
      name: rh-dev
```